Story/Episode 5/st_051080.csv

```diff
@@ -1,55 +1,55 @@
+st_051080_0040#0,"""...Selalu saja tiba-tiba.<br>Tapi aku senang kamu selamat."""
+st_051080_0050#0,"""Bisa beri tahu kami apa yang terjadi?"""
+st_051080_0060#0,"""Meski kau hanya bertanya, aku yakin kau hanya mengganggu mereka."""
+st_051080_0070#0,"""Aku minta maaf membawamu kembali secara tiba-tiba. Aku juga tidak sabar."""
+st_051080_0080#0,"""Dan kamu...<br>Bagaimana kamu bisa masuk ke sini?"""
+st_051080_0090#0,"""Bagaimana? Bukankah aku terhembus ke sini seperti angin?"""
+st_051080_0100#0,"""Oh! Kalian sedang mengadakan pertemuan rahasia? Maaf, aku harusnya mengetuk sebelum masuk, kan?"""
+st_051080_0110#0,"""Betapa kasarnya diriku ini. Mungkin karena aku telah sendiri sejak lama,<br>aku lupa bagaimana berinteraksi dengan orang lain."""
+st_051080_0120#0,"""Jejak teleportasi... tidak ada?<br>Dia bukan ARKS, tapi reaksi ini...<br>dia hanya terbuat dari Photon."""
+st_051080_0130#0,"""Tapi dia bukan Darker...<br>Apakah kamu tahu dia?"""
+st_051080_0140#0,"""Eh...<br>Kenapa semua orang bertingkah aneh?"""
+st_051080_0150#0,"""Kamu itu sebenarnya apa?"""
+st_051080_0160#0,"""Ah! Pantas. Kau tidak tahu siapa aku.<br>Maaf, hal itu tidak terlintas di pikiranku tadi."""
+st_051080_0170#0,"""Namaku Alma."""
+st_051080_0180#0,"""Apakah itu akan membantumu lebih mengerti kalau aku bilang bahwa aku adalah Claris Claes pertama?"""
+st_051080_0190#0,"""Namun saat ini, aku hanyalah Alma.<br>Akan lebih baik jika kalian bisa memanggilku Alma."""
+st_051080_0200#0,"""Claris Claes pertama?<br>Tapi dia seharusnya sudah tiada."""
+st_051080_0210#0,"""Tubuhku sudah tiada?<br>Ya, apa yang kalian lihat adalah Photon murni."""
+st_051080_0220#0,"""Seperti yang kalian tahu,<br>tubuhku telah dihancurkan oleh Luther."""
+st_051080_0230#0,"""Tapi hati dan Photon-ku tidak pernah hilang.<br>Mereka selalu berada dalam Akashic Record selama ini."""
+st_051080_0240#0,"""Singkatnya, ketika aku tewas, jiwaku menjadi bagian dari Akashic Record.<br>Mengerti?"""
+st_051080_0250#0,"""1. Kau ingin kami untuk percaya itu?"""
+st_051080_0260#0,"""2. Apakah kau bisa membuktikan itu?"""
+st_051080_0270#0,"""Pernyataan itu tadi?<br>Hmm, bagaimana supaya kalian bisa percaya aku..."""
+st_051080_0280#0,"""Hmm bukti, bukti... Apakah ada sesuatu... Oh iya!"""
+st_051080_0290#0,"""Ah, betul! Aku akan menunjukkan trik keren!"""
+st_051080_0300#0,"""Lihat dengan seksama.<br>Tiga, dua, satu..."""
+st_051080_0310#0,"""Eh? Kapan kamu...<br>Tadi itu bukan teleportasi?"""
+st_051080_0320#0,"""Sensasi itu... <%me> dan aku tahu sensasi itu."""
+st_051080_0330#0,"""Itu teknik yang sama dengan yang digunakan Luther untuk menghentikan waktu, kan?"""
+st_051080_0340#0,"""Iya. Dia berhasil menjiplak teknik itu<br>setelah melakukan beberapa eksperimen terhadapku."""
+st_051080_0350#0,"""Oh iya, Su-kun akan marah jika kalian mengklaim bahwa itu adalah Technique, dan bilang<br>\"Jangan bercanda. Ini sihir, bukan Technique.\""""
```

Comment:
Missing or stray quote in line 36 in st_051080.csv

Reply:
Yeah, I think I missed a quote line, let me change it quickly. Thanks!

Reply:
You KNOW you do not need to close the PR, you can add a commit to fix it, OR your can rewrite your git history to include the change `git rebase -i upstream/ID`

You may want to link CircleCI and TravisCI to your fork so you fix the issues before making a PR

Reply:
I see, thanks for the advice!

Reply:
Seems like after reopening the Pull Request, TravisCI and CircleCI doesn't perform any check on other Commits. Is this alright?